modules/lead/product-aws/aws.tf

```diff
@@ -1,31 +1,46 @@
 resource "aws_codebuild_project" "codebuild_build" {
   for_each = var.pipelines
 
-  name          = "${var.product_name}-${each.value.repo}-build"
+  name          = "${each.value.repo}-build"
   description   = "terraform_codebuild_project"
-  build_timeout = "5"
+  build_timeout = "20"
   service_role  = var.codebuild_role
 
   environment {
-    compute_type                = "BUILD_GENERAL1_SMALL"
-    image                       = "aws/codebuild/amazonlinux2-x86_64-standard:2.0"
+    privileged_mode             = true
+    compute_type                = "BUILD_GENERAL1_LARGE"
+    image                       = "489130170427.dkr.ecr.us-east-1.amazonaws.com/builder-image-skaffold:${var.builder_images_version}"
```

Comment:
we're using a variable called `toolchain_image_repo` in lead-environments that could be used here so we aren't hardcoding the ecr repo

Reply:
changed the value to be passed in as the toolchain_image_repo variable